README.md

```diff
@@ -15,7 +15,8 @@ Code structure:
 * `runnable.go` - contains a dummy implementation of the methods described in the design (`design/runnable.go`) for the runnable service, the actual implementation goes here.
 
 How to build and run:
-* run `make` to generate server, client and CLI code and build the `bin/fuseml_core` and `bin/fuseml_core-cli` binaries
+* run `make deps` to install go dependencies
+* run `make` to generate server, client and CLI code and build the `bin/fuseml_core` and `bin/fuseml_core-cli` binaries (NOTE: export variables for GITEA_URL for both client and server, and GITEA_USERNAME, GITEA_PASSWORD for server side)
```

Comment:
the NOTE should be in the next (run the binary) bullet IMO

Reply:
You are right indeed